image_db.py

```diff
@@ -6,8 +6,8 @@
 from datetime import datetime
 import logging
 
-TIME_FORMAT_NO_OFFESET = "%Y-%m-%d %H:%M:%S"
-TIME_FORMAT = TIME_FORMAT_NO_OFFESET + "%z"
+TIME_FORMAT_NO_OFFSET = "%Y-%m-%d %H:%M:%S"
```

Comment:
fixed what I think is not a deliberate a typo. changed to offset instead offeset.